rtt/base/TaskCore.cpp

```diff
@@ -337,5 +352,12 @@ namespace RTT {
         }
     }
 
+    std::string TaskCore::getName() const {
+        return name;
+    }
+    void TaskCore::setName(const std::string& name) {
+        this->name = name;
+        this->cName = strdup(name.c_str());
```

Comment:
Why does lltng-ust require this copy and we cannot simply replace all occurences of `cName` with `name.c_str()`?